src/infrastructure/logger/index.js

```diff
@@ -4,7 +4,7 @@ const winston = require('winston');
 const config = require('./../config');
 const appInsights = require('applicationinsights');
 const AppInsightsTransport = require('login.dfe.winston-appinsights');
-// const WinstonSequelizeTransport = require('login.dfe.audit.winston-sequelize-transport');
+ //const WinstonSequelizeTransport = require('login.dfe.audit.winston-sequelize-transport');
```

Comment:
remove the comments if not required

Reply:
this was declared but it was never used so I have removed thanks